extensions/binding/org.eclipse.smarthome.binding.ntp.test/pom.xml

```diff
@@ -0,0 +1,104 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.eclipse.smarthome.binding</groupId>
+  <artifactId>org.eclipse.smarthome.binding.ntp.tests</artifactId>
```

Comment:
org.eclipse.smarthome.binding.ntp.test  instead of org.eclipse.smarthome.binding.ntp.test**s**